packages/edit-site/src/index.js

```diff
@@ -88,15 +85,6 @@ export function initializeEditor( id, settings ) {
 
 	dispatch( editSiteStore ).updateSettings( settings );
 
-	// Keep the defaultTemplateTypes in the core/editor settings too,
-	// so that they can be selected with core/editor selectors in any editor.
-	// This is needed because edit-site doesn't initialize with EditorProvider,
-	// which internally uses updateEditorSettings as well.
-	dispatch( editorStore ).updateEditorSettings( {
-		defaultTemplateTypes: settings.defaultTemplateTypes,
-		defaultTemplatePartAreas: settings.defaultTemplatePartAreas,
-	} );
```

Comment:
Good riddance.

Reply:
Btw, I think there's some php code that we need to remove from the core as well (setting these in the editor setting) in the backport PR

Reply:
Yep, we can remove these lines:

https://github.com/WordPress/wordpress-develop/blob/1aa41dea3345c57cffce9059bbad728c86d1471a/src/wp-admin/site-editor.php#L64-L65